static/app/views/alerts/rules/details/body.tsx

```diff
@@ -412,6 +406,12 @@ const DetailWrapper = styled('div')`
   }
 `;
 
+const StatusWrapper = styled('div')`
+  display: flex;
+  align-items: center;
+  gap: ${space(0.5)};
```

Comment:
I don't know that we can support this yet https://caniuse.com/?search=flex%20gap

Reply:
Mostly just because of safari